### PR TITLE
FAQ for multi-project builds

### PR DESCRIPTION
**Pull request by advayDev1**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -13,6 +13,41 @@ due to support for native compilation features.
 You have to first create the [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html).
 Go to your project folder and do ``gradle wrapper``. Refresh your Eclipse project.
 
+### How do I setup multiple related j2objc or native projects?
+You can express three kinds of dependencies within j2objcConfig:
+
+1.  j2objc project B depends on the j2objc library in project A.  This normally corresponds
```
Comment (brunobowden) [inline FAQ.md]: Reword:
The common case is that Java Project A depends on Java Project B... so you want the the Project A J2ObjC generated library to depend on the Project B J2ObjC generated library". In this case add to B.gradle:

File: FAQ.md [hunk 2]
```diff
@@ -13,6 +13,41 @@ due to support for native compilation features.
...
+to the Java classes in B depending on A as well.  In this case add to B.gradle:
+    ```
+    j2objcConfig {
+        dependsOnJ2objc project(':A')
+    }
+    ```
+This kind of dependency should be inferred automatically from the corresponding Java
+dependency in the future.
```
Comment (brunobowden) [inline FAQ.md]: Good. Glad you added that in there.

File: FAQ.md [hunk 3]
```diff
@@ -13,6 +13,41 @@ due to support for native compilation features.
...
+
+2.  j2objc project B depends on a
+[custom native library](https://docs.gradle.org/current/userguide/nativeBinaries.html#N15F82)
+L in project A.  Add to B.gradle:
```
Comment (brunobowden) [inline FAQ.md]: What's the "L" at the start of the line?

Comment (brunobowden) [inline FAQ.md]: Explain what "L" is for as I'm not familiar with it.

File: FAQ.md [hunk 4]
```diff
@@ -13,6 +13,41 @@ due to support for native compilation features.
...
+    ```
+    j2objcConfig {
+        extraNativeLib project: ':A', library: 'L', linkage: 'static'
+    }
+    ```
+
+3.  j2objc project B depends on library libA pre-built outside of Gradle in directory /lib/SOMEPATH,
+with corresponding headers in /inc/SOMEPATH.  Add to B.gradle:
```
Comment (brunobowden) [inline FAQ.md]: Fully expand inc => include to make that clearer. I think that "lib" is common enough to be familiar.

File: FAQ.md [hunk 5]
```diff
@@ -13,6 +13,41 @@ due to support for native compilation features.
...
+2.  j2objc project B depends on a
+[custom native library](https://docs.gradle.org/current/userguide/nativeBinaries.html#N15F82)
+L in project A.  Add to B.gradle:
+    ```
+    j2objcConfig {
+        extraNativeLib project: ':A', library: 'L', linkage: 'static'
+    }
+    ```
+
+3.  j2objc project B depends on library libA pre-built outside of Gradle in directory /lib/SOMEPATH,
```
Comment (brunobowden) [inline FAQ.md]: libA => libPreBuilt.a so that it's clearer

Also modify the linker flags below to -lPreBuilt

**Comment by brunobowden** [comment]
LGTM

File: FAQ.md (hunk 3 above)
Comment (advayDev1) [inline FAQ.md]: clearly the math variable format is not readable :)

L is the name of a library.  i'll make this less formal

File: FAQ.md (hunk 1 above)
Comment (advayDev1) [inline FAQ.md]: ok

File: FAQ.md (hunk 5 above)
Comment (advayDev1) [inline FAQ.md]: done

File: FAQ.md (hunk 4 above)
Comment (advayDev1) [inline FAQ.md]: sure

**Pull request merged**